### PR TITLE
update frontend files

### PR DESCRIPTION
**Pull request by RosarioAngelica**
Please review my code. Thank youu!

### CODE REVIEW

File: rosario_angelica/design.css [hunk 1]
```diff
@@ -173,4 +173,73 @@ textarea:focus {
...
+}
+
+.section {
+  flex: 1;
+  margin: 10px;
+  background-color: #FEFDED;
+  border-radius: 10px;
+  padding: 20px;
+  box-shadow: 4px 5px 15px black;
+}
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/design.css:185]: Separate every css block with a single space

File: rosario_angelica/design.css [hunk 2]
```diff
@@ -173,4 +173,73 @@ textarea:focus {
...
+.comment-section {
+  max-width: 600px;
+  margin: auto;
+  background: white;
+  padding: 20px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/design.css]:
```suggestion
.section {
  flex: 1;
  margin: 10px;
  background-color: #FEFDED;
  border-radius: 10px;
  padding: 20px;
  box-shadow: 4px 5px 15px black;
}

.comment-section {
  max-width: 600px;
  margin: auto;
  background: white;
  padding: 20px;
  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
}
```

Comment (Odrunia-Mark-Ryan) [reply rosario_angelica/design.css]: Separate every css block with a single space

File: rosario_angelica/script.js [hunk 3]
```diff
@@ -1,12 +1,88 @@
...
-    commentInput.value.trim().length > 0) {
-    submitBtn.disabled = false;
-  } else {
-    submitBtn.disabled = true;
+document.addEventListener('DOMContentLoaded', () => {
+  const commentForm = document.getElementById('comment-Form');
+  const commentText = document.getElementById('comment-Text');
+  const commentsList = document.getElementById('comments-List');
+  const sortAsc = document.getElementById('sort-Asc');
+  const sortDesc = document.getElementById('sort-Desc');
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/script.js]: use the document.querySelector here instead of getElementById

File: rosario_angelica/script.js [hunk 4]
```diff
@@ -1,12 +1,88 @@
...
+  saveComments();
+});
+
+sortDesc.addEventListener('click', () => {
+  comments.sort((a, b) => new Date(b.date) - new Date(a.date));
+  displayComments();
+  saveComments();
+});
+
+// Load comments on page load
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/script.js]: remove unnecessary comment

File: rosario_angelica/design.css [hunk 5]
```diff
@@ -173,4 +173,73 @@ textarea:focus {
...
+button {
+  padding: 10px 15px;
+  background-color: #ff5e40;
+  color: white;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+button:hover {
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/design.css]: add a space after the colon "button: hover"

**Review by Odrunia-Mark-Ryan — CHANGES_REQUESTED** [review]

File: rosario_angelica/script.js [hunk 6]
```diff
@@ -1,88 +1,69 @@
...
 
-  let comments = [];
+  if (
+    input_field.value.trim().length === 0 ||
+    comment_field.value.trim().length === 0
+  ) {
+    submit_btn.disabled = true;
+    submit_btn.classList.add("disabled");
+    return;
+  }
```
Comment (Odrunia-Mark-Ryan) [inline rosario_angelica/script.js]:
```suggestion
if (!input_field.value.trim() || !comment_field.value.trim()) {
  submit_btn.disabled = true;
  submit_btn.classList.add("disabled");
  return;
}

```

Comment (Odrunia-Mark-Ryan) [reply rosario_angelica/script.js]: you can optimize this code and make your code much shorter

**Review by Odrunia-Mark-Ryan — CHANGES_REQUESTED** [review]

**Review by Odrunia-Mark-Ryan — APPROVED** [review]

**Comment by RosarioAngelica** [comment]
Thank you!

**Pull request merged**